lib/patch/define-property.js

```diff
@@ -1,11 +1,14 @@
 'use strict';
 
+var utils = require('../utils');
+
 // might need similar for object.freeze
 // i regret nothing
 
 var _defineProperty = Object.defineProperty;
 var _getOwnPropertyDescriptor = Object.getOwnPropertyDescriptor;
 var _create = Object.create;
+var unconfigurablesKey = utils.getKey('unconfigurables');
```

Comment:
Good point, I'll factor this out into `keys.js`. utils already has a fair bit of stuff in it.